SWIG/swap.i

```diff
@@ -166,6 +168,69 @@ class VanillaSwapPtr : public SwapPtr {
     }
 };
 
+%{
```

Comment:
You don't need the additional class; you can expose `MakeVanillaSwap` directly.
See in [scheduler.i](https://github.com/lballabio/QuantLib-SWIG/blob/master/SWIG/scheduler.i) how it was done for `MakeSchedule`. (Also, the same file shows how we can give it a more natural interface in Python.)